tests/integration/kubernetes/confidential_common.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+# Copyright 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+function setup_unencrypted_confidential_pod() {
+	get_pod_config_dir
+	
+	export SSH_KEY_FILE="${pod_config_dir}/confidential/unencrypted/ssh/unencrypted"
+	
+	# Use correct address in pod yaml
+	sed -i "s/-nightly/-${PR_NUMBER}/" "${pod_config_dir}/pod-confidential-unencrypted.yaml"
```

Comment:
I think the issue still stands? If I run it locally then `$PR_NUMBER=""` , so the setup replace would end up creating `ghcr.io/kata-containers/test-images:unencrypted-`? What about changing it to
```suggestion
   if [ -n "${PR_NUMBER}" ]; then
      sed -i "s/-nightly/-${PR_NUMBER}/" "${pod_config_dir}/pod-confidential-unencrypted.yaml"
   fi
```
which I think should solve that, or have I missed something?

Reply:
That's fair, but I think that if you're running things locally you'll face more issues than that related to the PR_NUMBER, as that's also used as part of the kata-deploy image being used.  For instance: https://github.com/kata-containers/kata-containers/pkgs/container/kata-deploy-ci/122932726?tag=7778-ee9acee96dd74391cd876f8fb07b5f63eed05461-amd64

I'm not saying this is *not* a problem, but I'm rather saying that we better solve this as part of https://github.com/kata-containers/kata-containers/pull/7621 than as part of this one.

Reply:
Ok that's fair enough. I know that Wainer is very keen for things to be able to run locally, but if it's tracked elsewhere, then I'm happy to not hold this up

Reply:
Looking here, it seems that PR_NUMBER is only used in two places:
* kata-deploy imagine, whuch can be easily overriden with the work Wainer did on #7621 
* this test, which would be a problem.

I'm taking your suggestion, @stevenhorsman, as it's only one case and if we can avoid keeping the problem here, better.